testing/regress/ecl/parquet_partition.ecl

```diff
@@ -0,0 +1,68 @@
+/*##############################################################################
+    HPCC SYSTEMS software Copyright (C) 2024 HPCC Systems®.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//class=parquet
+//fail
+
+IMPORT Std;
+IMPORT Parquet;
+
+// Define the record layout for the dataset
+datasetRecordLayout := RECORD
+    INTEGER id;
+    STRING name;
+    INTEGER age;
+    STRING city;
+END;
+
+// Create a small dataset
+smallData := DATASET([
+    {1, 'Alice', 30, 'New York'},
+    {2, 'Bob', 25, 'Los Angeles'},
+    {3, 'Charlie', 40, 'Chicago'}
+], datasetRecordLayout);
+
+// Set options
+overwriteOption := TRUE;
+rowSize := 1;
+
+// Write out the dataset with Hive partitioning on CITY
+ParquetIO.HivePartition.Write(
+    smallData,
+    rowSize,                            // Number of rows per file
+    '/var/lib/HPCCSystems/mydropzone/hive_partitioned/',
```

Comment:
Use Std.File.GetDefaultDropZone() + '/regress/parquet/hive_partitioned/' to avoid cluttering up the dropzone root directory and assuming the platform is installed in root.

Reply:
For this file, I have fixed it in terms of the partitioning I believe, but this is the current error that I am still trying to figure out : assert(!"Unknown copy source type") failed - file: hqlcpp.cpp, line 11772

Reply:
That doesn't look like an issue with the plugin. Perhaps your ECL is still incorrect. I noted a few issues in my comments.